linear/linear.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("The program solves a linear inequality Ax + B > 0.")
+	fmt.Print("Enter A and B: ")
+	var a, b float64
+	fmt.Scan(&a, &b)
+	if a == 0 && b == 0 {
+		fmt.Println("No solutions.")
+	} else if a == 0 {
+		fmt.Println("No solutions.")
+	} else if b == 0 {
```

Comment:
I don't think you should take `b == 0` as a special case. It's *exactly* the same as non-0.